Automattic-Tracks-iOS/Model/Core Data/TracksContextManager.m

```diff
@@ -3,10 +3,25 @@
 
 @interface TracksContextManager ()
 
+@property (nonatomic, assign) BOOL sandboxed;
+
 @end
 
 @implementation TracksContextManager
 
+- (instancetype)init {
+    return [[TracksContextManager alloc] initWithSandboxedMode:true];
```

Comment:
This initializer returning a new instance, rather than the one being initialized.

You would typically instantiate a Context Manager like this:

```
TracksContextManager *manager = [[TracksContextManager alloc] init];
```

1. `alloc` reserves memory for the object
2. `init` is expected to initialize the instance

In this case, `init` is alloc'ating more memory, and returning yet another instance.

Reply:
Thank you for this! Gosh 😳 I'm really rusty at writing Objective-C 🤦 

I felt like there was something wrong with that, but it compiled and the tests passed so I shrugged it off.

Interestingly, Xcode also gives me a warning about it

![image](https://user-images.githubusercontent.com/1218433/94876763-fca44c80-049b-11eb-9fde-d0c732f77a53.png)

It either came after I wrote the code and moved on or after I added the `NS_DESIGNATED_INITIALIZER` annotation. Regardless, I shouldn't have missed it... 😳